scripts/start/projects/project_WetHorizons.R

```diff
@@ -0,0 +1,132 @@
+# |  (C) 2008-2025 Potsdam Institute for Climate Impact Research (PIK)
+# |  authors, and contributors see CITATION.cff file. This file is part
+# |  of MAgPIE and licensed under AGPL-3.0-or-later. Under Section 7 of
+# |  AGPL-3.0, you are granted additional permissions described in the
+# |  MAgPIE License Exception, version 1.0 (see LICENSE file).
+# |  Contact: magpie@pik-potsdam.de
+
+# ----------------------------------------------------------
+# description: Project WetHorizons Peatland
+# position: 5
+# ----------------------------------------------------------
+
+## Load lucode2 and gms to use setScenario later
+library(lucode2)
+library(gms)
+library(gdx2)
+library(magpie4)
+
+# Load start_run(cfg) function which is needed to start MAgPIE runs
+source("scripts/start_functions.R")
+
+# Source default cfg. This loads the object "cfg" in R environment
+source("config/default.cfg")
+
+# create additional information to describe the runs
+cfg$info$flag <- "WH01"
+
+cfg$results_folder <- "output/:title:"
+cfg$results_folder_highres <- "output"
+cfg$force_replace <- TRUE
+cfg$force_download <- FALSE
+cfg$qos <- "standby_dayMax"
+
+# support function to create standardized title
+.title <- function(cfg, ...)
+  return(paste(cfg$info$flag, sep = "_", ...))
+
+cfg$repositories <- append(
+  list(
+    "https://rse.pik-potsdam.de/data/magpie/public" = NULL,
+    "./patch_inputdata" = NULL
+  ),
+  getOption("magpie_repos")
+)
+
+cfg$gms$c_timesteps <- "5year"
+ssp <- "SSP2"
+
+# Reference
+cfg$title <- .title(cfg, paste(ssp, "Ref", sep = "-"))
+cfg <- setScenario(cfg, c(ssp, "NPI", "rcp7p0"))
+cfg$gms$c56_mute_ghgprices_until <- "y2150"
+cfg$gms$c56_pollutant_prices <- paste0("R32M46-", if (ssp == "SSP2") "SSP2EU" else ssp, "-NPi")
+cfg$gms$c60_2ndgen_biodem    <- paste0("R32M46-", if (ssp == "SSP2") "SSP2EU" else ssp, "-NPi")
```

Comment:
A bit nicer to make sure (also makes it easier to see) the two settings are actually set to the same:
```suggestion
cfg$gms$c60_2ndgen_biodem    <- cfg$gms$c56_pollutant_prices
```

Reply:
With the actual name ("R32M46-SSP2EU-NPi") it is easy to see that the two settings are identical.